packages/web-app-files/src/views/trash/Overview.vue

```diff
@@ -0,0 +1,244 @@
+          </template>
+          <template #name="{ item }">
+            <oc-button
+              class="oc-display-block"
+              type="router-link"
+              appearance="raw"
+              :to="getTrashLink(item)"
+              v-text="getSpaceName(item)"
+            />
+          </template>
+          <template #footer>
+            <div class="oc-text-nowrap oc-text-center oc-width-1-1 oc-my-s">
+              <p class="oc-text-muted">{{ footerTextTotal }}</p>
+              <p v-if="filterTerm" class="oc-text-muted">{{ footerTextFilter }}</p>
+            </div>
+          </template>
+        </oc-table>
+      </div>
+    </files-view-wrapper>
+  </div>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, nextTick, onMounted, ref, unref, watch } from 'vue'
+import { buildSpace } from 'web-client/src/helpers'
+import Mark from 'mark.js'
+import Fuse from 'fuse.js'
+import { useGettext } from 'vue3-gettext'
+import { useTask } from 'vue-concurrency'
+import {
+  configurationManager,
+  useCapabilityProjectSpacesEnabled,
+  useGraphClient,
+  useRouter
+} from 'web-pkg'
+import { createLocationTrash } from 'web-app-files/src/router'
+import { createFileRouteOptions } from 'web-pkg/src/helpers/router'
+import AppBar from 'web-app-files/src/components/AppBar/AppBar.vue'
+import FilesViewWrapper from 'web-app-files/src/components/FilesViewWrapper.vue'
+
+export default defineComponent({
+  name: 'TrashOverview',
+  components: { FilesViewWrapper, AppBar },
+  methods: { createLocationTrash },
+  setup: function () {
+    const router = useRouter()
+    const { $gettext } = useGettext()
+    const sortBy = ref('name')
+    const sortDir = ref('asc')
+    const filterTerm = ref('')
+    const markInstance = ref(undefined)
+    const tableRef = ref(undefined)
+    const spaces = ref([])
+    const { graphClient } = useGraphClient()
+    const hasProjectSpaces = useCapabilityProjectSpacesEnabled()
+
+    const loadResourcesTask = useTask(function* (signal) {
+      const {
+        data: { value: drivesResponse }
+      } = yield unref(graphClient).drives.listMyDrives()
+      const drives = drivesResponse
+        .map((space) => buildSpace({ ...space, serverUrl: configurationManager.serverUrl }))
+        .filter(
+          (space) =>
+            space.disabled !== true &&
+            (space.driveType === 'project' || space.driveType === 'personal')
+        )
+
+      spaces.value = orderBy(drives, 'name', 'asc')
+    })
+
+    const footerTextTotal = computed(() => {
+      return $gettext('%{spaceCount} trashes in total', {
+        spaceCount: unref(spaces).length.toString()
+      })
+    })
+    const footerTextFilter = computed(() => {
+      return $gettext('%{spaceCount} matching trashes', {
+        spaceCount: unref(orderedSpaces).length
+      })
+    })
+
+    const breadcrumbs = computed(() => [
+      { text: $gettext('Deleted files'), onClick: () => loadResourcesTask.perform() }
```

Comment:
```suggestion
      { text: $gettext('Deleted files') }
```

Just omit the click handler. No need to reload the spaces.

Reply:
Lets do it like in projects.vue

Reply:
For the record: `like in projects.vue` means that we reload the project spaces on breadcrumb item click, so that the user has a chance to reload the spaces without reloading the page.